test/DMLTest.php

```diff
@@ -468,4 +468,13 @@ public function test_18_02_delete_loop() {
         gc_collect_cycles();
         gc_collect_cycles();
     }
+
+	/**
+	 * @doesNotPerformAssertions
+	 */
+	public function test_19_schema_obtain_failed() {
+        $port = TestHelpers::getTarantoolPort();
+		$empty_tarantool = new Tarantool('localhost', $port, 'test');
```

Comment:
The problem appears only when there is no fetched schema. It is not obvious from the testing code and should be clarified in a comment.

Reply:
Clarified.